notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Could you elaborate a bit more about Decoder-Only model?</p><p>I think it's good to mention how it is different from the encoder-decoder approach in terms of text generation, and touch some actual model names (e.g. T5 vs GPTs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Absolutely! Added an explanation along with link to their origin papers.</p>